test/dlc_tests/container_tests/bin/pytorch_tests/testPyTorchwithInductorHelper

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+HOME_DIR=/test
+BIN_DIR=${HOME_DIR}/bin
+LOG_DIR=${HOME_DIR}/logs
+
+EXAMPLESDIR=${HOME_DIR}/artifacts/examples
+set -e
+cd ${EXAMPLESDIR}
+
+TRAINING_LOG=${LOG_DIR}/pytorch_train_bert_with_inductor.log
+
+pip install deepspeed==0.8.2
+
+echo "Training BertForMaskedLM using PyTorch dynamo and inductor backend... This may take a few minutes. You can follow progress on the log file : $TRAINING_LOG"
+set +e
+python benchmarks/dynamo/huggingface.py --only=BertForMaskedLM --float32 -dcuda --training --inductor --performance 2>&1 | tee $TRAINING_LOG
```

Comment:
What is the intention here ? This command currently only runs 1 forward pass and 1 backward pass. This is not a test of end to end training.

Reply:
Got it. I will pick a more end-to-end training script.